src/lib/safe-log.ts

```
@@ -0,0 +1,53 @@
+ */
+export function errDetail(error: unknown): string {
+  const base = errMessage(error);
+  const response = (error as { response?: { status?: number; data?: unknown } })?.response;
+  if (!response?.status) return base;
+
+  const apiMessage =
+    typeof response.data === 'object' && response.data !== null
+      ? (
+          (response.data as { error?: { message?: string } | string }).error as
+            | { message?: string }
+            | undefined
```

ESLint

[error] Replace `|·{·message?:·string·}⏎···········` with `{·message?:·string·}`

(prettier/prettier)